src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2796,6 +2796,10 @@ static bool createEntity
       {
         oeP->fill(SccInvalidModification, "Already Exists", "Unprocessable");
       }
+      else if (errDetail->find("Can't extract geo keys") != std::string::npos)
```

Comment:
An entry describing the change done in this PR should be added to CHANGES_NEXT_RELEAES. Something like this:

```
- Fix: return 400 instead of 500 when GeoJSON syntax error occur (#3347)
```

Reply:
Fixed in ae914b6